targets/UnrealMarketplacePlugin/makeBp.js

```diff
@@ -296,3 +298,29 @@ function getAuthBools(tabbing, apiCall) {
 
     return output;
 }
+
+function getCustomApiAssignmentLogic(tabbing, apiCall) {
+    if (apiCall.name === "ExecuteFunction")
```

Comment:
I don't think this is a strict enough check. api.name and apiCall.name would be specific enough.